Detections/Corelight/CorelightForcedExternalOutboundSMB.yaml

```diff
@@ -0,0 +1,29 @@
+id: 73f23aa2-5cc4-4507-940b-75c9092e9e01
+name: Corelight - Forced External Outbound SMB
+description: |
+  'Detects SMB requests that originate internally and communicate with an external IP address.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1187
+query: |
+  Corelight
+  | where EventType =~ 'conn'
+  | where ZeekConnLocalSrc == 'True'
+  | where ZeekConnLocalDst == 'False'
+  | where NetworkConnectionHistory startswith 'Sh' or NetworkApplication startswith 'smb'

```

Comment:
Let's use hasprefix here, it won't help performance on the 'Sh' matching but it will on the 'smb' matching - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/datatypes-string-operators

Reply:
updated